hooks/pre_build_centos

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -ex
+yum install epel-release -y
+# yum update -y
+# our deps
+yum install distgen make -y
+
+mkdir -p root/
+make VARIANT=centos DISTRO=centos-7-x86_64 source
```

Comment:
Probably needs to be `make TARGET=centos7 generate`? We also do not need the `root/` directory creation.